### PR TITLE
Major update to make working with BP bench easier with modern kotlin

### PR DESCRIPTION
**Pull request by scribe**
@nicholasbvolpe This PR is for your work

### CODE REVIEW

**Review by david-limbach — DISMISSED** [review]
LGTM

File: gradle.properties [hunk 1]
```diff
@@ -1,6 +1 @@
-clikt=1.7.0
-kotlinGuiced=0.0.5
-kotlinxCoroutines=1.4.3
-ds3=5.0.4
-mockk=1.9.3.kotlin12
-kotlintest=3.3.2
\ No newline at end of file
+ds3=5.0.4
```
Comment (lesserwhirls) [inline gradle.properties]: My only suggestion would be to put the version in `build.gradle.kts` which would let us shed `gradle.properties` all together.

Comment (scribe) [reply gradle.properties]: That's a good idea, let me switch over to that.

Comment (scribe) [reply gradle.properties]: Should be fixed.

**Review by david-limbach — APPROVED** [review]

**Review by lesserwhirls — APPROVED** [review]

**Comment by lesserwhirls** [comment]
Looks like we're unstuck now :-)

**Review by david-limbach — APPROVED** [review]

**Pull request merged**